ocp_resources/node_network_configuration_policy.py

```diff
@@ -103,6 +104,7 @@ def __init__(
         self.state = state or self.Interface.State.UP
         self.set_ipv4 = set_ipv4
         self.set_ipv6 = set_ipv6
+        self.success_timeout = success_timeout
```

Comment:
Please remove it from here, the function `wait_for_status_success` should get it.

Reply:
Can't supply it `to wait_for_status_success` if an nncp is deployed using `with`, unless passed on __init__